js/src/common/components/Modal.js

```diff
@@ -103,7 +112,7 @@ export default class Modal extends Component {
    * Hide the modal.
    */
   hide() {
-    this.attrs.onhide();
+    this.attrs.state.close();
```

Comment:
The naming and usage of `hide`, `onhide` and `close` is now very irritating, IMO.

Reply:
Agree, but not that much we can do. Renamed onhide to animateHide

Reply:
Shouldn't `onshow` also be renamed in that case? It looks inconsistent in `ModalManager` line 17 now.